src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -107,21 +107,19 @@ private function createRelationsForItem($item, $formattedRelations)
             $relationType = $relationDetails['relation_type'];
 
             switch ($relationType) {
-                case 'BelongsTo':
-                    $modelInstance = $relationDetails['model']::find($relationDetails['values'])->first();
-                    if ($modelInstance != null) {
-                        $relation->associate($modelInstance)->save();
-                    } else {
-                        $relation->dissociate()->save();
-                    }
-                    break;
                 case 'HasOne':
                 case 'MorphOne':
-                        $modelInstance = $this->createUpdateOrDeleteOneToOneRelation($relation, $relationMethod, $relationDetails);
+                        if (is_null($relationDetails['values'])) {
```

Comment:
Basically moved the function over here, but removed some things that covered cases that we don't have anymore.